src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,130 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    static class Node<K, V> {
+        private final int hash;
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(int hash, K key, V value, Node<K, V> next) {
```

Comment:
what about access modifiers?(go through all the code and add them where they are missing)

Reply:
Why you don't fix this comment? Constructors also need to specify access modifiers

Reply:
Sorry missed that